tests/testsuite/script.rs

```diff
@@ -1107,79 +1046,81 @@ fn cmd_metadata_with_embedded() {
-                "links": null,
-                "description": null,
-                "readme": null,
-                "repository": null,
-                "rust_version": null,
-                "homepage": null,
-                "documentation": null,
-                "homepage": null,
-                "documentation": null,
-                "targets": [
-                    {
-                        "kind": [
-                            "bin"
-                        ],
-                        "crate_types": [
-                            "bin"
-                        ],
-                        "doc": true,
-                        "doctest": false,
-                        "test": true,
-                        "edition": "[..]",
-                        "name": "script",
-                        "src_path": "[..]/script.rs"
-                    }
-                ],
-                "features": {},
-                "manifest_path": "[..]script.rs",
-                "metadata": null,
-                "publish": []
-            }
-        ],
-        "workspace_members": ["path+file:[..]foo#script@0.0.0"],
-        "workspace_default_members": ["path+file:[..]foo#script@0.0.0"],
-        "resolve": {
-            "nodes": [
-                {
-                    "dependencies": [],
-                    "deps": [],
-                    "features": [],
-                    "id": "path+file:[..]foo#script@0.0.0"
-                }
-            ],
-            "root": "path+file:[..]foo#script@0.0.0"
-        },
-        "target_directory": "[ROOT]/home/.cargo/target/[..]",
-        "version": 1,
-        "workspace_root": "[..]/foo",
-        "metadata": null
-    }"#,
+      "authors": [],
+      "categories": [],
+      "default_run": null,
+      "dependencies": [],
+      "description": null,
+      "documentation": null,
+      "edition": "[..]",
+      "features": {},
+      "homepage": null,
+      "id": "path+[ROOTURL]/foo#script@0.0.0",
+      "keywords": [],
+      "license": null,
+      "license_file": null,
+      "links": null,
+      "manifest_path": "[ROOT]/foo/script.rs",
+      "metadata": null,
+      "name": "script",
+      "publish": [],
+      "readme": null,
+      "repository": null,
+      "rust_version": null,
+      "source": null,
+      "targets": [
+        {
+          "crate_types": [
+            "bin"
+          ],
+          "doc": true,
+          "doctest": false,
+          "edition": "[..]",
+          "kind": [
+            "bin"
+          ],
+          "name": "script",
+          "src_path": "[ROOT]/foo/script.rs",
+          "test": true
+        }
+      ],
+      "version": "0.0.0"
+    }
+  ],
+  "resolve": {
+    "nodes": [
+      {
+        "dependencies": [],
+        "deps": [],
+        "features": [],
+        "id": "path+[ROOTURL]/foo#script@0.0.0"
+      }
+    ],
+    "root": "path+[ROOTURL]/foo#script@0.0.0"
+  },
+  "target_directory": "[ROOT]/home/.cargo/target/[..]",
```

Comment:
Does this no longer need `[..]`?

Reply:
Great catch! I've just updated the regex to allow directories that don't end with a slash.